blockbase/theme.json

```diff
@@ -535,7 +535,7 @@
 					"fontFamily": "var(--wp--preset--font-family--heading-font)",
 					"fontWeight": "var(--wp--custom--heading--typography--font-weight)",
 					"lineHeight": "var(--wp--custom--heading--typography--line-height)",
-					"fontSize": "32px"
+					"fontSize": "var(--wp--preset--font-size--huge)"
```

Comment:
is this coming from elsewhere or just a fix?

Reply:
It's just a fix. I'll create a separate PR for it.

Reply:
https://github.com/Automattic/themes/pull/5101